docs/getting-started-on-snowplow-bdp-enterprise/setup-guide-azure/index.md

```diff
@@ -1,8 +1,86 @@
 ---
 title: "Setup Snowplow BDP on Azure"
+date: "2024-01-11"
 sidebar_position: 30
 ---
 
-## Coming soon
+## Request your pipeline through Snowplow BDP Console
 
-If you are interested in deploying [BDP Enterprise](https://snowplow.io/pricing/) in your Azure account, please [join our waiting list](https://snowplow.io/bdp-enterprise-on-azure/).
+Login to Snowplow BDP Console where you'll be able to follow a step-by-step guide to getting started (including the steps below).
+
+## Setting up your Azure account
+
+To get your cloud environment ready for your Snowplow pipeline to be installed:
+
+### Create a new account
+
+Please create a new Azure account and tenant segregated from anything else you have running in Azure. For the latest documentation on setting up a new account and tenant please refer to the Azure documentation [here](https://azure.microsoft.com/).
```

Comment:
```suggestion
Please create a new Azure account and tenant segregated from anything else you have running in Azure. For the latest documentation on setting up a new account and tenant please refer to the [Azure documentation](https://azure.microsoft.com/).
```

Reply:
Is there any specific page we could link to?

Reply:
I've been unable to find any generic "Create your azure account" type documentation from them. Most of it is bare bones and sends you right to signing up with Azure. I did find this https://azure.microsoft.com/en-us/pricing/purchase-options/pay-as-you-go/ which has a bit more detail. Let me know if you'd like to update to using this or leave.